vendor.conf

```diff
@@ -23,3 +23,7 @@ golang.org/x/oauth2 1611bb46e67abc64a71ecc5c3ae67f1cbbc2b921
 golang.org/x/sys 99f16d856c9836c42d24e7ab64ea72916925fa97
 google.golang.org/api 1202890e803f07684581b575fda809bf335a533f
 google.golang.org/grpc 0713829b980f4ddd276689a36235c5fcc82a21bf
+github.com/aws/aws-sdk-go fa107560b5f3528a859a1a1511086646731bb1a8
+github.com/go-ini/ini afbc45e87f3ba324c532d12c71918ef52e0fb194
+github.com/jmespath/go-jmespath bd40a432e4c76585ef6b72d3fd96fb9b6dc7b68d
+cloud.google.com/go 0bee953ffbf6fda3be3f56f1e9578f16a1e978a4
```

Comment:
I had some issues with vendoring. By just adding the AWS deps I managed to remove 90% of the other dependencies 😢 Perhaps it was an old `vndr` or perhaps it was issues running locally vs in a container. Either way, I've fixed it now and will push an update